From 747345de8e438ca936b313b1de34b79f656a2327 Mon Sep 17 00:00:00 2001
From: effects3d <effects3dee@gmail.com>
Date: Sun, 13 Sep 2026 13:47:44 +0200
Subject: [PATCH 07/11] =?UTF-8?q?Revert=20=E2=80=9CHACK:=20winex11:=20Disa?=
 =?UTF-8?q?ble=20host=20window=20decorations=20with=20Mutter.=E2=80=9D?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Roblox Studio doesn’t change its window decorations at any point in
time, so it’s best to revert this patch to avoid having a Win95-esq
title bar on GNOME.

Source: Proton-Wine commit: 20bf0d401d65e51b50d8e9e137230d7eff2179ca
Signed-off-by: effects3d <effects3dee@gmail.com>
---
 dlls/winex11.drv/window.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/dlls/winex11.drv/window.c b/dlls/winex11.drv/window.c
index cc3b6185b70..e9716e25821 100644
--- a/dlls/winex11.drv/window.c
+++ b/dlls/winex11.drv/window.c
@@ -534,8 +534,6 @@ static unsigned long get_mwm_decorations_for_style( DWORD style, DWORD ex_style
 {
     unsigned long ret = 0;
 
-    if (X11DRV_HasWindowManager( "Mutter" )) return 0;
-
     if (ex_style & WS_EX_TOOLWINDOW) return 0;
     if (ex_style & WS_EX_LAYERED) return 0;
 
-- 
2.55.0

